### DISCUSSION TITLE
Possibility to see the "Published At" date?

### DISCUSSION THREAD

**Discussion by perostlin** [payloadcms/payload#2492] [Feature Requests & Ideas]
Like the other dates, Last Modified and Created, possibility to see the "Published At" date as well?
This is something we are currently missing in our current CMS, Strapi, to be able to tell from the Admin UI when a entity was published, not just last modified.

**Sidenote**:
When updating an entity the "Last Modified" field does not update until page reload, can create repro if needed.

<img width="356" alt="image" src="https://user-images.githubusercontent.com/20464832/231830844-99b6c54f-8129-4a28-b033-cb0eff802411.png">

**Comment by KevinYobeth** [comment]
is there an update to this? I'm making use of this function for a blog collection, I want to display the published at date, yet I can't seem to query or get it from the versioning collections

**Reply by r1tsuu** [reply]
While there's no "out of the box" solution, it's easy to add such functionallity via hooks.

```ts
 {
      admin: {
        readOnly: true,
      },
      hooks: {
        beforeValidate: [
          (args) => {
            if (!args.data || args.value || args.originalDoc?.publishedAt) return;
            if (args.data._status === 'published') return new Date().toISOString();
          },
        ],
      },
      name: 'publishedAt',
      type: 'date',
    },
```
